SONAR-9566 Issues at org level

gregaubert opened the pull request.

stas-vilchik commented on server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.js:
```diff
@@ -35,6 +36,7 @@ const ADMIN_PATHS = [
 
 export default class OrganizationNavigation extends React.PureComponent {
   props: {
+    currentUser: { isLoggedIn: boolean, showOnboardingTutorial: true },
```
remove `showOnboardingTutorial`

gregaubert replied:
+1

stas-vilchik commented on tests/src/test/java/org/sonarqube/tests/issue/OrganizationIssuesPageTest.java:
```diff
@@ -0,0 +1,96 @@
+import static util.ItUtils.restoreProfile;
+import static util.ItUtils.runProjectAnalysis;
+
+public class OrganizationIssuesPageTest {
```
add this class to `tests/src/test/java/org/sonarqube/tests/Category6Suite.java`

gregaubert commented:
Thanks @stas-vilchik, I removed the organization name in breadcrumbs and and the project facet in the last commit.
As for the "My Issues" filter I just discussed with Fabrice because it seemed to weird to me to remove it just here (we have it at project level) and we agreed to keep the "My Issues" at organization level too. I updated the MMF.

gregaubert commented:
@stas-vilchik Fixed in commit 79659e4, thanks!
